app/src/examples/WithClampExample.tsx

```diff
@@ -0,0 +1,114 @@
+import Animated, {
+  useSharedValue,
+  useAnimatedStyle,
+  withSpring,
+  withClamp,
+} from 'react-native-reanimated';
+import { View, Text, Button, StyleSheet, ViewStyle } from 'react-native';
+import React, { useState } from 'react';
+
+/**  TODO #5239 As soon as 'withClamp' is included in reanimated version used in docs
```

Comment:
Personally I like the idea of giving github :octocat:  PR number here, because
- If you don't know why sb added this TODO comment you can quicker find a valid PR
- Some of TODO comments are duplicated in multiple files, since the change should affect several files. You can easily search in code for all the TODOs created in given PR to check if you should address more of them.